### PR TITLE
Update docker env set up

### PR DESCRIPTION
**Pull request by lthurston**

### CODE REVIEW

**Comment by lthurston** [comment]
@amywieliczka Here's that docker env only branch, teased apart from the ucd branch.

File: docker/.ipython/profile_default/startup/bootstrap_fetcher.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python
+
+import os
+import glob
+import importlib
+import sys
```
Comment (amywieliczka) [inline docker/.ipython/profile_default/startup/bootstrap_fetcher.py:6]:
```suggestion
```

Comment (amywieliczka) [reply docker/.ipython/profile_default/startup/bootstrap_fetcher.py:6]: make ruff happy

File: docker/.ipython/profile_default/startup/bootstrap_mapper.py [hunk 2]
```diff
@@ -3,19 +3,19 @@
 import os
 import glob
 import importlib
-# from pprint import pprint
+import sys
```
Comment (amywieliczka) [inline docker/.ipython/profile_default/startup/bootstrap_mapper.py:6]:
```suggestion
```

File: docker/.ipython/profile_default/startup/README [hunk 3]
```diff
@@ -0,0 +1 @@
+There's some DRY violations in here. It'd be nice to fix them, but would add some weird complexity.
```
Comment (amywieliczka) [inline docker/.ipython/profile_default/startup/README:1]: Readability > DRY 🙏 thank you

**Review by amywieliczka — CHANGES_REQUESTED** [review]
There's some RUFF complaints in here, but otherwise looks good thanks so much for breaking this out.

**Review by amywieliczka — APPROVED** [review]

**Pull request merged**